Updates How to Send and Receive SMS Messages With Node.js and Express

amdcavallaro opened the pull request:
Updates the code for[How to Send and Receive SMS Messages With Node.js and Express](https://learn.vonage.com/blog/2019/09/16/how-to-send-and-receive-sms-messages-with-node-js-and-express-dr/).

**Changes**

1. Removes bodyParser.
2. Updates Readme.
3. Updates Nexmo reference to Vonage.
4. Updates .env.example.

amdcavallaro committed:
Updates code

conshus commented on README.md:
```diff
@@ -1,3 +1,20 @@
+## Further Reading
+
+* Check out the Developer Documentation at <https://developer.vonage.com>
+* Details about Vonage SMS Functionality https://developer.vonage.com/messaging/sms/overview
```
```suggestion
* Details about Vonage SMS Functionality <https://developer.vonage.com/messaging/sms/overview>
```

conshus commented on autoresponder.js:
```diff
@@ -1,25 +1,21 @@
-
+const Vonage = require("@vonage/server-sdk");
+const express = require("express");
 const app = require('express')()
```
Pretty sure you can do this also since you extracted `express` as a variable. It's what we did in our app-to-phone tutorial
```suggestion
const app = express();
```

conshus commented on autoresponder.js:
```diff
@@ -1,25 +1,21 @@
 });
 
-var text = "👋Hello from Nexmo";
+var text = "👋Hello from Vonage";
```
Should this be moved inside of app.post() since it's the only place it's used and just declare it since it's being set later? Also, since we are updating this, do we want to use const and let?
```suggestion
var text ;
```

conshus commented on server.js:
```diff
@@ -1,10 +1,8 @@
-const app = require('express')()
-const bodyParser = require('body-parser')
+const express = require("express");
+const app = require('express')();
```
```suggestion
const app = express();
```

conshus reviewed (commented):
I think it looks good. Just a couple of comments.

amdcavallaro committed:
Changes as requested in PR

amdcavallaro committed:
Moves text declaration inside of post

amdcavallaro commented:
Thanks a mil for the review @conshus, this has been super helpful!

amdcavallaro committed:
Adds ;

juliacodez committed:
Initial commit

Modifying repo to use the Vonage Messages API only + applying modern JavaScript

juliacodez committed:
Merge branch 'update_autoresponder' into update-to-messages-only

juliacodez committed:
Fix Vonage version

juliacodez committed:
Update package.json

Commit:
Merge pull request #2 from nexmo-community/update-to-messages-only

Update repo to use Messages API only
